docs/cardano/liquid-staking/c1-overview/stakedmilkada.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: "StakedMilkADA"
+sidebar_position: 3
+---
+
+You can find the full contract [here]([https://github.com/dcSpark/liquid-staking](https://github.com/dcSpark/liquid-staking/blob/main/src/stakedMilkAda/StakedMilkAda.sol)).
+
+# StakedMilkAda (stMilkAda)
```

Comment:
You'll want to modify the `contract-overview` file instead of this

Reply:
I have updated the `contract-overview` file, and deleted the `stakedmilkada.mdx` file.